frontend/src/component/admin/network/NetworkTrafficUsage/chart-functions.ts

```diff
@@ -15,9 +15,11 @@ export type ChartDatasetType = ChartDataset<'bar'>;
 
 export const toTrafficUsageChartData = (
     traffic: TrafficUsageDataSegmentedCombinedSchema,
+    filter?: string,
```

Comment:
adjusted a function used for data usage with an option to filter out only one traffic type e.g. frontend